lib/Backends/Interpreter/tests/InterpreterBackendCorrectnessTest.cpp

```diff
@@ -17,4 +17,9 @@
 
 using namespace glow;
 
-std::set<std::string> glow::backendTestBlacklist = {};
+std::set<std::string> glow::backendTestBlacklist = {
+#ifndef GLOW_WITH_CPU
+    // Requires the CPU target due to the use of MockCPUBackend.
+    "dataParallelStackingTest/0",
```

Comment:
Good catch! It seems like this test needs a backend supporting the kernel stacking functionality. Currently only the CPU backend supports this. IMHO, other backends should  unconditionally exclude it.

Reply:
I made this blacklisting unconditional.